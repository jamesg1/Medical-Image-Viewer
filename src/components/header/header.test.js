import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Header from './header';

describe('Header', () => {
  it('renders component correctly with title', () => {
    const wrapper = shallow(<Header title="Medical Image Series Viewer" />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  it('renders component without props', () => {
    const wrapper = shallow(<Header />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
