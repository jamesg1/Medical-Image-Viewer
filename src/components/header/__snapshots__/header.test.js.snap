// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders component correctly with title 1`] = `
<header
  className="header"
>
  <img
    alt="logo"
    className="App-logo"
    src="logo.svg"
  />
  <h1
    className="header-title"
  >
    Medical Image Series Viewer
  </h1>
</header>
`;

exports[`Header renders component without props 1`] = `
<header
  className="header"
>
  <img
    alt="logo"
    className="App-logo"
    src="logo.svg"
  />
  <h1
    className="header-title"
  />
</header>
`;
